core-services/prow/02_config/openshift/cluster-api-provider-vsphere/_prowconfig.yaml

```diff
@@ -0,0 +1,70 @@
+tide:
+  queries:
+  - includedBranches:
+    - openshift-4.9
+    - release-4.9
```

Comment:
We are really close to GA which will change the merge criteria. If this PR merges *before* the GA day, then we just need to regenerate https://github.com/openshift/release/pull/26761. 

If this merges *after*, then this file should be changes. This query needs to include the 4.10 items, the `staff-eng-approved` one will need to change to 4.11, and the one with `excludedBranches` will need 4.11 items added, too.